app/javascript/css/tailwind.css

```diff
@@ -21,5 +24,11 @@ h1 {
     grid-template-rows: 150px 150px;
     grid-row-gap: 30px;
     grid-column-gap: 50px;
-    @apply justify-center mt-6;
+    @apply justify-center;
+    @apply  mt-6;
 }
+
+body {
+    background-color: #f4f4f4;
```

Comment:
Use tailwind to apply the background color and add that color `#f4f4f4` to a variable in tailwind.js

Reply:
done